src/provider/repository/package.rs

```diff
@@ -0,0 +1,41 @@
+use std::convert::TryFrom;
+use std::fmt;
+
+use oci_distribution::Reference;
+use serde::{Deserialize, Serialize};
+
+use crate::provider::error::StackableError;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Package {
+    pub product: String,
+    pub version: String,
+}
+
+impl Package {
+    pub fn get_file_name(&self) -> String {
+        format!("{}.tar.gz", self.get_directory_name())
+    }
+
+    pub fn get_directory_name(&self) -> String {
```

Comment:
These two methods could use some documentation what it is they return, it's not clear from the name